SimCalorimetry/HcalSimProducers/src/HcalDigitizer.cc

```diff
@@ -30,7 +30,7 @@
 #include <boost/foreach.hpp>
 #include <memory>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
This should be commented back

Reply:
There is #46708 with the fix.

Reply:
> There is #46708 with the fix.

Please either profit of that PR, or make a new one dedicated, to fix also the comment line as [above](https://github.com/cms-sw/cmssw/pull/46570#discussion_r1844930269)